src/ontology/mondo-edit.obo

```diff
@@ -146641,8 +146641,8 @@ xref: MESH:C537554 {source="Orphanet:2538/e", source="MONDO:equivalentTo", sourc
 xref: OMIM:156810 {source="Orphanet:2538/e", source="MONDO:equivalentTo", source="Orphanet:2538"}
 xref: Orphanet:2538 {source="MONDO:equivalentTo", source="OMIM:156810"}
 xref: UMLS:C1834929 {source="Orphanet:2538/e", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="Orphanet:2538", source="OMIM:156810"}
+is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432"} ! syndromic disease
 is_a: MONDO:0015160 {source="Orphanet:2538"} ! multiple congenital anomalies/dysmorphic syndrome-variable intellectual disability syndrome
-is_a: MONDO:0015210 {source="Orphanet:2538"} ! syndromic gastroduodenal malformation
```

Comment:
should this term stay in the "digestive tract malformation" branch? (MONDO:0020019)
It is a syndrome with "small stomach (microgastria) "

Reply:
we haven't necessarily been classifying syndromes as children of all the features. If we made this a child of 'digestive tract malformation', then it should also be a child of something that would include 'limb reduction defect' too. I've been classifying terms like this as hereditary diseases and/syndromic diseases for the most part. (And there are some that are classified under these other Orphanet grouping terms, like here. We should probably revisit this grouping class and the related ones later.